client/src/interface/User.ts

```diff
@@ -1,8 +1,11 @@
+import { Profile } from './Profile';
+
 export interface User {
   email: string;
   username: string;
   _id: string;
   id: string;
```

Comment:
Why do we have two id's in our User interface?

Reply:
Concerning the '_id' i did not add that. Probably Hassan did.

Reply:
@rajivtitus

Reply:
I was unable to access profile of because the controllers are set up as such.